src/configuration.c

```diff
@@ -0,0 +1,56 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include "../include/configuration.h"
+#include "../include/utils.h"
+/*
+    Gibt SUCCESS oder eine Fehlermeldung über 1000 zurück
+*/
+enum Position {key, value};
+
+const char PORT_KEY[] = "PORT";
+const char PATH_KEY[] = "PATH";
+const char MAX_SESSIONS_KEY[] = "MAX_SESSIONS";
+
+Configuration config = {5678, "./data", 5};
+
+int loadConfig(char path[]) {
+    char c;
+    FILE *fp;
+    fp = fopen(path, "r");
+    if (fp == NULL) {
+        printf("An error occurred while opening the config file. The application will stick to its default config.\n");
+        return 1001;
+    }
+    char keyContent[STRING_LENGTH] = "";
+    char valueContent[STRING_LENGTH] = "";
```

Comment:
Still not the PATH_MAX length. This will not introduce any bugs, because the Configuration.PATH string is longer than this string, but when PATH_MAX chars get allocated, it would be nice if PATH_MAX is also the real usable amount.